deploy/helm/sumologic/conf/setup/setup.sh

```diff
@@ -15,8 +15,28 @@ if [[ ${DEBUG_MODE,,} == ${DEBUG_MODE_ENABLED_FLAG} ]]; then
     done
 fi
 
-# Fix URL to remove "v1" or "v1/"
-export SUMOLOGIC_BASE_URL=${SUMOLOGIC_BASE_URL%v1*}
+function fix_sumo_base_url() {
+  local BASE_URL=$SUMOLOGIC_BASE_URL
+
+  if [[ ! "$BASE_URL" =~ ^https:\/\/.*sumologic\.com\/api\/.*$ ]]; then
```

Comment:
We should just check if this is set instead of making sure if it's a sumologic.com subdomain. It's valid to put other urls here for testing - this is why integration tests are currently failing.